chapters/synchronous.tex

```diff
@@ -1103,59 +1113,60 @@ \section{Clocked Discretized Partition}\label{continuous-time-equations-in-clock
+Therefore, outside the discretized partions themselves, they are treated similarly to discrete-time partitions.
+Especially, operators such as \lstinline!sample!, \lstinline!hold!, \lstinline!subSample! must be used to communicate signals of the discretized partition with other partitions.
 
-\subsection{Clocked Discrete-Time and Discretized Continuous-Time}\label{clocked-discrete-time-and-clocked-discretized-continuous-time-partition}
+\subsection{Discrete-time and Discretized Partitions}\label{clocked-discrete-time-and-clocked-discretized-continuous-time-partition}\label{clocked-and-discretized-partition}
 
 Additionally to the variability of expressions defined in \cref{variability-of-expressions}, an orthogonal concept \willintroduce{clocked variability} is defined in this section.
 If not explicitly stated otherwise, an expression with a variability such as \emph{continuous-time} or \emph{discrete-time} means that the expression is inside a partition that is not associated to a clock.
 If an expression is present in a partition that is not a continuous-time partition, it is a \firstuse[clocked!expression]{clocked expression}\index{expression!clocked} and has \firstuse[clocked!variability]{clocked variability}\index{expression variability!clocked}.
 
-After sub-clock inferencing, see \cref{sub-clock-inferencing}, every partition that is associated to a clock has to be categorized as \willintroduce{clocked discrete-time} or \willintroduce{clocked discretized continuous-time} partition.
+After sub-clock inferencing, see \cref{sub-clock-inferencing}, every sub-clock partition that is associated with a clock has to be categorized as a \willintroduce{discrete-time} or a \willintroduce{discretized} partition.
+\begin{nonnormative}
+Previously, discrete-time partition was refered to as \emph{clocked discrete-time partition}, and discretized partition as \emph{clocked discretized continuous-time partition}.
+\end{nonnormative}
 
-If a clocked partition contains no operator \lstinline!der!, \lstinline!delay!, \lstinline!spatialDistribution!, no event related operators from \cref{event-related-operators-with-function-syntax} (with exception of \lstinline!noEvent! and \lstinline!smooth!), and no \lstinline!when!-clause with a \lstinline!Boolean! condition, it is a \firstuse[clocked!discrete-time partition]{clocked discrete-time}\index{partition!clocked discrete-time} partition.
+If a sub-clock partition that is not continuous-time contains any of the operators \lstinline!der!, \lstinline!delay!, \lstinline!spatialDistribution!, or event related operators from \cref{event-related-operators-with-function-syntax} (with exception of \lstinline!noEvent! and \lstinline!smooth!), or contains a \lstinline!when!-clause with a \lstinline!Boolean! condition, it is a \firstuse[discretized!partition]{discretized}\index{partition!discretized} partition.
```

Comment:
With the risk of mixing up the roles of base partitions and sub-clock partitions, it's better to be explicit here (also giving better symmetry with the next sentence):
```suggestion
If a sub-clock partition that is not continuous-time contains any of the operators \lstinline!der!, \lstinline!delay!, \lstinline!spatialDistribution!, or event related operators from \cref{event-related-operators-with-function-syntax} (with exception of \lstinline!noEvent! and \lstinline!smooth!), or contains a \lstinline!when!-clause with a \lstinline!Boolean! condition, it is a \firstuse[discretized!partition]{discretized}\index{partition!discretized} sub-clock partition.
```